packages/api/src/api/index.ts

```diff
@@ -12,6 +14,8 @@ if (env.nodeEnv !== 'development') api.use(enforceRateLimiting);
 
 // UNPROTECTED ROUTES
 api.use('/health', health);
+api.use('/auth', auth);
+api.use('/callback', callback);
```

Comment:
This should moved into the auth base route

Reply:
So `/callback` should be moved into `api/auth/index.ts`?

Reply:
It should be registered in that file so the full url would be `/api/auth/callback/slack`